drools-ansible-rulebook-integration-api/src/test/java/org/drools/ansible/rulebook/integration/api/AssignmentTest.java

```diff
@@ -264,4 +265,201 @@ public void testAssignmentReferenceListContainsItemExpressionRHS() {
+                                "condition": {
+                                    "AllCondition": [
+                                        {
+                                            "AssignmentExpression": {
+                                                "lhs": {
+                                                    "Events": "varname"
+                                                },
+                                                "rhs": {
+                                                    "EqualsExpression": {
+                                                        "lhs": {
+                                                            "Event": "xyz"
+                                                        },
+                                                        "rhs": {
+                                                            "Integer": 300
+                                                        }
+                                                    }
+                                                }
+                                            }
+                                        },
+                                        {
+                                            "SelectExpression": {
+                                                "lhs": {
+                                                    "Events": "varname.levels"
+                                                },
+                                                "rhs": {
+                                                    "operator": {
+                                                        "String": ">"
+                                                    },
+                                                    "value": {
+                                                        "Integer": 25
+                                                    }
+                                                }
+                                            }
+                                        }
+                                    ]
+                                },
+                                "actions": [
+                                    {
+                                        "Action": {
+                                            "action": "echo",
+                                            "action_args": {
+                                                "message": "Hurray"
+                                            }
+                                        }
+                                    }
+                                ],
+                                "enabled": true
+                            }
+                        }
+                    ]
+                }
+                """;
+
+        assertThatThrownBy(() -> {
+            RulesExecutorFactory.createFromJson(JSON);
+        })
+                .as("SelectExpression doesn't support inversion")
+                .isInstanceOf(UnsupportedOperationException.class);
```

Comment:
At the moment, we don't inverse `ConditionFactory` operators (`ExistsField`, `SelectConstraint`, `SelectAttrConstraint`, `SearchMatchesConstraint`). We may consider them one by one if requested.

For now, we explicitly throw an Exception.